constellations/clients/orionFakeDataFiller/filler.go

```diff
@@ -228,28 +238,87 @@ func runFiller(hostAddress string) {
 	)
 
 	// Create sessions
-	createSession(
+	createSessions(
 		hostAddress,
-		"id_1",
+		"ap_java_2020_fall_class1",
 		"false",
+		3,
 	)
 
-	createSession(
+	createSessions(
 		hostAddress,
-		"id_2",
+		"ap_calculus_2020_fall_class1",
 		"true",
+		2,
+	)
+
+	// Create Accounts
+	createAccount(
+		hostAddress,
+		"emailaddress1@gmail.com",
+		"jhdgjhddjhdjuj",
+	)
+	createAccount(hostAddress,
+		"emailaddress2@gmail.com",
+		"2redssssa",
+	)
+
+	// Create Users
+	createUser(
+		hostAddress,
+		"Joe",
+		"Smith",
+		"",
+		"emailaddress1@gmail.com",
+		"301-123-4567",
+		0,
+		1,
+		"notes1",
+		"schoolone",
+		2001,
+	)
+
+	createUser(
+		hostAddress,
+		"Billy",
+		"Bob",
+		"Joe",
+		"emailaddress2@gmail.com",
+		"301-123-4568",
+		1,
+		2,
+		"notes2",
+		"schooltwo",
+		2002,
+	)
+
+	// Create transactions
+	createTransaction(
+		hostAddress,
+		100,
+		"pay_paypal",
+		"notes1",
+		1,
+	)
+
+	createTransaction(
+		hostAddress,
+		101,
+		"pay_cash",
+		"notes2",
+		2,
```

Comment:
add one more transaction that's a charge (don't mind which one)

Reply:
it would be great if one of these accounts had a negative a balance (i.e. stills owe us)